gsmphys/satmedmfvdifq.f

```diff
@@ -438,6 +449,17 @@ subroutine satmedmfvdifq(ix,im,km,ntrac,ntcw,ntiw,ntke,
            kcld(i)  = km1
          endif
       enddo
+!           
+! compute a function for green vegetation fraction and surface roughness
+!       
+      do i = 1,im
+        !tem = (sigmaf(i) - vegflo) / (vegfup - vegflo)
+        !tem = min(max(tem, 0.), 1.)
+        !tem1 = sqrt(tem)
+        !ptem = (z0(i) - z0lo) / (z0up - z0lo)
+        !ptem = min(max(ptem, 0.), 1.)
+        vez0fun(i) = 1. !(1. + vc0 * tem1) * (1. + zc0 * ptem)
```

Comment:
tem1 and ptem are commented out here and values from a previous loop will be used to calculate `vez0fun`